requirements/cython.in

```diff
@@ -1,3 +1,3 @@
--r multidict.txt
+multidict==5.2.0
```

Comment:
`.in` file deps should usually be unpinned with the constraints kept in `.txt` files. The main reason for other entries to keep the pins was that these were corner cases IIRC.